lisa/notifiers/json.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import json
+from dataclasses import dataclass
+from datetime import datetime
+from decimal import Decimal
+from enum import Enum
+from typing import Any, Dict, List, Type, cast
+
+from dataclasses_json import dataclass_json
+
+from lisa import constants, messages, notifier, schema
+
+
+@dataclass_json()
+@dataclass
+class JsonSchema(schema.Notifier):
+    path: str = "lisa.json"
+
+
+class Json(notifier.Notifier):
```

Comment:
It just focuses on perf messages, and dump a better format to consume by other tools. It should be like List[Dict[str, Any]] format. All messages are dumped in the same array.

Reply:
updated